tests/unit/amo/api/test_users.js

```diff
@@ -50,7 +50,7 @@ describe(__filename, () => {
     });
   });
 
-  describe('editUserAccount', () => {
+  describe('updateUserAccount', () => {
     const getParams = (params = {}) => {
       const state = dispatchSignInActions().store.getState();
       const userId = getCurrentUser(state.users).id;
```

Comment:
There's a line below (line 61), which should be changed from `it('edits a userProfile and...` to `it('updates a userProfile and...`.

Reply:
This test description still needs to be updated in `tests/unit/amo/api/test_users.js `.